sdk/search/search-documents/README.md

```diff
@@ -84,39 +81,122 @@ const serviceClient = new SearchServiceClient("<endpoint>", new AzureKeyCredenti
 
-`SearchServiceClient` provides methods for configuring and customizing an Azure Cognitive Search instance. The client currently has support for creating and managing search indexes and will later expand to support creating and managing other service entities such as indexers, synonym maps, cognitive skillsets, and data sources.
+## Examples
 
-**Note**: This client cannot function in the browser because the APIs it calls do not have support for Cross-Origin Resource Sharing (CORS).
+### Create an Index
 
-### Documents
+```js
+const { SearchServiceClient, AzureKeyCredential } = require("@azure/search-documents");
 
-An item stored inside a search index. The shape of this document is described in the index using `Field`s. Each Field has a name, a datatype, and additional metadata such as if it is searchable or filterable.
+const client = new SearchServiceClient("<endpoint>", new AzureKeyCredential("<apiKey>"));
 
-### Pagination
+async function main() {
+  const result = await client.createIndex({
+    name: "example-index",
+    fields: [
+      {
+        type: "Edm.String",
+        name: "id",
+        key: true
+      },
+      {
+        type: "Edm.Double",
+        name: "awesomenessLevel",
+        sortable: true,
+        filterable: true,
+        facetable: true
+      },
+      {
+        type: "Edm.String",
+        name: "description",
+        searchable: true
+      },
+      {
+        type: "Edm.ComplexType",
+        name: "details",
+        fields: [
+          {
+            type: "Collection(Edm.String)",
+            name: "tags",
+            searchable: true
+          }
+        ]
+      },
+      {
+        type: "Edm.Int32",
+        name: "hiddenWeight",
+        hidden: true
+      }
+    ]
+  });
 
-Typically you will only wish to [show a subset of search results](https://docs.microsoft.com/azure/search/search-pagination-page-layout#total-hits-and-page-counts) to a user at one time. To support this, you can use the `top`, `skip` and `includeTotalResultCount` parameters to provide a paged experience on top of search results.
+  console.log(result);
+}
 
-### Document field encoding
+main();
+```
 
-[Supported data types](https://docs.microsoft.com/rest/api/searchservice/Supported-data-types) in an index are mapped to JSON types in API requests/responses. The JS client library keeps these mostly the same, with some exceptions:
+### Upload documents into an index
 
-- `Edm.DateTimeOffset` is converted to a JS `Date`.
-- `Edm.GeographyPoint` is converted to a `GeographyPoint` type exported by the client library.
-- Special values of the `number` type (NaN, Infinity, -Infinity) are serialized as strings in the REST API, but are converted back to `number` by the client library.
```

Comment:
How are developers to know what we will automatically encode and decode types for them in their custom models?

Reply:
wouldn't this kind of the detail be covered in the API reference?

Reply:
API reference being our generated reference docs? 

I worry this kind of detail would be hard to find and I'm not sure where we would put it exactly.